java/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -59,6 +59,9 @@
   "name":"double",
   "allDeclaredMethods":true
 },
+{
+  "name":"groovy.lang.Binding"
```

Comment:
Why do we need this ? Hades2 integration has no Groovy dependency? Are you sure it is necessary?